components/calendar/src/iso.rs

```diff
@@ -306,7 +306,7 @@ impl DateTime<Iso> {
                 + i32::from(self.time.hour.number()) * minutes_a_hour
                 + i32::from(self.time.minute.number())
         } else {
-            0
+            -1
```

Comment:
Suggestion: I think this line is unreachable, so consider putting an `unreachable!()` here